rules/rename-teams-channel-folder/rule.md

```diff
@@ -26,25 +26,27 @@ There is a workaround to rename the folder correctly.
 
   ::: greybox
   **Note:** This hasn't changed the name of the folder in SharePoint!
-  ![Figure: The folder name didn't change yet](renamingachannel3.png)
-  :::
+
+
+
+![Figure: The folder name didn't change yet](blob:https://ssw.com.au/a2c62bbf-aa13-4adc-90e4-f15319319a7d)
```

Comment:
Hey @Marxoz 

The image path has been changed here which I believe will cause a 404.

1. Can you fix?
2. What are the goals of this PR? (I can't tell by looking at the changes)

Reply:
Hey @tiagov8,

The image was not appearing in th firs place. It was only being referenced by an URL and not inside an image object. I placed it into an image object, and the preview seemed to show that it works

I did not make any changes to the URL though

Reply:
@tiagov8 

This was the issue I was trying to fix:![image](https://user-images.githubusercontent.com/86134557/212573685-80b6bb02-8dd6-45f9-8fc6-bc5fd45d9ea8.png)

Reply:
Thanks for clarifying @Marxoz 

It's weird the image URL has changed... Anyways, I've used your PR to include the necessary changes to fix the greyboxes. Should be all good. Please have a look at the Markdown with my changes and let me know if in doubt ;)